app/dashboard/views.py

```diff
@@ -25,20 +25,27 @@
 from django.conf import settings
 from django.contrib import messages
 from django.contrib.auth.models import User
+from django.core.paginator import EmptyPage, PageNotAnInteger, Paginator
```

Comment:
Should be able to use `paginator.get_page` vs manually catching these cases now.